public/js/views/transaction/complete-payment.jsx

```diff
@@ -42,16 +42,19 @@ export default class CompletePayment extends Component {
   }
 
   render() {
+    var email = (this.props.userEmail ?
+      <p className="receipt">
+        {gettext('Your receipt has been sent to')}
+        <span className="email">{this.props.userEmail}</span>
+      </p> : '');
+
     return (
       <div className="complete">
         <ProductDetail productId={this.props.productId} />
         <p className="accepted">{gettext('Payment Accepted')}</p>
-        <p className="receipt">
-          {gettext('Your receipt has been sent to')}
-          <span className="email">{this.props.userEmail}</span>
-        </p>
```

Comment:
I think `{email}` should go here, not below the submit button to follow the mocks